Sprint-2/debug/address.js

```diff
@@ -12,4 +18,4 @@ const address = {
   postcode: "XYZ 123",
 };
 
-console.log(`My house number is ${address[0]}`);
+console.log(`My house number is ${address.houseNumber}`);
```

Comment:
Good work. Make sure to keep your comments formatted the same way, for example have a space after "//" for better readability.

Reply:
Thanks for reviewing my code. yes sure